consensus/src/config.rs

```diff
@@ -78,27 +88,48 @@ pub fn exponential_slowdown(
 
 /// A default configuration of what the creators of this package see as optimal parameters.
 pub fn default_config(n_members: NodeCount, node_ix: NodeIndex, session_id: SessionId) -> Config {
-    let unit_creation_delay = Arc::new(|t| {
-        if t == 0 {
-            Duration::from_millis(5000)
-        } else {
-            exponential_slowdown(t, 500.0, 3000, 1.005)
-        }
-    });
-    let delay_config = DelayConfig {
-        tick_interval: Duration::from_millis(100),
-        requests_interval: Duration::from_millis(3000),
-        unit_rebroadcast_interval_min: Duration::from_millis(15000),
-        unit_rebroadcast_interval_max: Duration::from_millis(20000),
-        // 4000, 8000, 16000, 32000, ...
-        unit_creation_delay,
-        // 5000, 500, 500, 500, ... (till step 3000), 500, 500*1.005, 500*(1.005)^2, 500*(1.005)^3, ..., 10742207 (last step)
-    };
     Config {
         node_ix,
         session_id,
         n_members,
-        delay_config,
+        delay_config: DelayConfig {
+            tick_interval: Duration::from_millis(100),
```

Comment:
If I remember correctly the scheduling of tasks has precision `+/- tick_interval` hence we should probably decrease it to be like `10ms`, otherwise we might have delays of around `200ms` instead of the planned `100ms`.

Reply:
I think this is still unresolved?

Reply:
I did decrease the timeouts to `0, 50, ...`, so it should in practice always be "on next tick". I think changing the default `tick_interval` has potentially far reaching consequences and would like to avoid it - going from `100` to `10` would basically mean the worker is woken up 10x more often, right? WDYT?

Reply:
Btw. this is the default config, so it doesn't matter THAT MUCH, since it is not used in aleph-node anyway, but just wanted to emphasize that having some timeouts larger than `tick_interval` is probably not ideal.